### PR TITLE
Adds `bubblewrap fingerprint`

### PR DESCRIPTION
**Pull request by andreban**
- Adds the fingerprint command that allows:
  - adding keys.
  - removing keys.
  - listing keys.
  - generating assetlinks.json.

- Removes assetlinks.json generation from `build`

### CODE REVIEW

**Review by joycetoh8 — COMMENTED** [review]
This works great! Some minor suggestions and also thoughts:
- Should the `add` subcommand prevent adding fingerprints that have already been added?
- Just an idea and doesn't have to be a part of this PR, but for the `remove` subcommand I think it could also be useful to allow removing by the name. But then there could be the issue of if they named multiple fingerprints the same name.
- What's the thought process behind removing the assetlinks.json generation during `build`? I think it could still be useful to generate it for the local signing key if someone wants to just deploy locally and test without going through Play first.

**Review by PEConn — COMMENTED** [review]
Still need to have a good look through `fingerprint.ts`, but leaving this one comment here for noq.

File: packages/cli/src/lib/inputHelpers.ts [hunk 1]
```diff
@@ -254,3 +254,18 @@ export async function validatePackageId(input: string): Promise<Result<string, E
...
+/**
+ * A {@link ValidateFunction} that receives a {@link string} as input and resolves to a
+ * {@link string} when successful. Verifies if the input is a valid SHA-256 fingerprint.
+ * @param {string} input a string representing a SHA-256 fingerprint.
+ * @returns {Result<string, Error>} a result that resolves to a {@link string} on
+ * success or {@link Error} on failure.
+ */
+export async function validateSha256Fingerprint(input: string): Promise<Result<string, Error>> {
+  input = input.toUpperCase();
+  if (input.match(/([0-9-A-F]{2}:?){32}/)) {
```
[the remark this answers is not in the document]
Comment (andreban) [reply packages/cli/src/lib/inputHelpers.ts]: That's right, thanks for catching it! Adjusted.

**Review by PEConn — APPROVED** [review]
This all looks good, thanks Andre!

**Pull request merged**